ci-operator/step-registry/rhoai/patch-component-deployment/odh-model-controller/rhoai-patch-component-deployment-odh-model-controller-commands.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -o verbose
+
+namespace="redhat-ods-applications"
```

Comment:
can't we make the namespace as parameter so we could reuse this for ODH as well?

Reply:
This step is currently under `rhoai` and is used in a flow which installs RHOAI.
Once there's additional support to install ODH in this flow, the step can be moved and extended.